packages/engine/src/worker/runner/python/mod.rs

```diff
@@ -27,7 +26,8 @@ use crate::{
 };
 
 pub struct PythonRunner {
-    init_msg: Arc<ExperimentInitRunnerMsg>, // Args to RunnerImpl::new
+    init_msg: Arc<ExperimentInitRunnerMsg>,
+    // Args to RunnerImpl::new
```

Comment:
This doesn't look right

Reply:
I wonder why `rustfmt` didn't change this before...